src/utils/useGeolocation.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from 'react';
+
+interface locationType {
+  loaded: boolean;
+  coordinates?: { lat: number; lng: number };
```

Comment:
좌표값이 무조건 필요하니까, ?로 있어도 되고 없어도 된다는 표시를 제거해주세요.
그리고, 변수명을 줄여서 사용하지말고 그대로 사용하죠. 브라우저에서 제공하는 기능이니까 그게 좀 더 직관적입니다.
```suggestion
   coordinates: {
      latitude: number;
      longitude: number;
   }
```